src/screens/ChoosePinScreen.js

```diff
@@ -18,15 +18,18 @@ import { t } from 'ttag';
 import NewHathorButton from '../components/NewHathorButton';
 import HathorHeader from '../components/HathorHeader';
 import PinInput from '../components/PinInput';
-import { setInitWallet, unlockScreen } from '../actions';
+import { startWalletRequested, unlockScreen } from '../actions';
 import { PIN_SIZE } from '../constants';
 
 import baseStyle from '../styles/init';
 
 
 const mapDispatchToProps = (dispatch) => ({
   unlockScreen: () => dispatch(unlockScreen()),
-  setInitWallet: (words, pin) => dispatch(setInitWallet(words, pin)),
+  startWalletRequested: (words, pin) => dispatch(startWalletRequested({
```

Comment:
This reducer is not doing anything with the payload, why do you send it?

Reply:
The reducer is not doing anything with it, but the `startWallet` saga is